tests/e2e/koperator_suite_test.go

```diff
@@ -55,8 +55,9 @@ var _ = BeforeSuite(func() {
 
 var _ = When("Testing e2e test altogether", Ordered, func() {
 	//testInstall()
-	//testProduceConsumeInternal()
-	testUninstallZookeeperCluster()
+	testInstallZookeeperCluster(1)
+	testInstallKafkaCluster("local", "simplekafkacluster.yaml")
```

Comment:
The SSL cluster install would basically be the same, just with the filename change to `simplekafkacluster_ssl.yaml`

Reply:
**Required**: IMO we **MUST** add both variants here.